Common/src/CConfig.cpp

```diff
@@ -3651,20 +3678,70 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
     }
   }
 
+  if (Molecular_Weight == nullptr){
+    Molecular_Weight = new su2double[1];
+    Molecular_Weight[0] = 28.96; 
```

Comment:
There is a bit of hardcoding going here. Even for developing purposes it might be good if we tackle that